deepdish/io/hdf5io.py

```diff
@@ -10,20 +10,25 @@
 except ImportError:
     _pandas = False
 
+try:
+    from types import SimpleNamespace as sns
```

Comment:
That's a good point. Our little group uses that, but I doubt it's common practice. I'll just use SimpleNamespace.